deblur/deblurring.py

```diff
@@ -0,0 +1,167 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from operator import attrgetter
+
+import numpy as np
+
+from deblur.sequence import Sequence
+
+
+def get_sequences(input_seqs):
+    """Returns a list of Sequences
+
+    Parameters
+    ----------
+    input_seqs : iterable of (str, str)
+        The list of input sequences in (label, sequence) format
+
+    Returns
+    -------
+    list of Sequence
+
+    Raises
+    ------
+    ValueError
+        If no sequences where found in `input_seqs`
+        If all the sequences do not have the same length either aligned or
+        unaligned.
+    """
+    seqs = [Sequence(label, seq) for label, seq in input_seqs]
```

Comment:
Okay, I see, it does get tricky due the pairwise calculation step. What is the expected upper bound for the number of sequences needed to be held in memory?